includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1483,6 +1479,17 @@ public function process_payment_for_order( $cart, $payment_information, $schedul
 		];
 	}
 
+	/**
+	 * The parent method which allows to modify the child class implementation, while supporting the current design where the parent process_payment method is called from the child class.
+	 * Mandate must be shown and acknowledged under certain conditions for Stripe Link and SEPA.
+	 * Since WC_Payment_Gateway_WCPay represents card payment, which does not require mandate, we return false.
+	 *
+	 * @return boolean False since card payment does not require mandate.
+	 */
+	protected function does_payment_method_require_mandate_data() {
```

Comment:
(minor) just as an alternative with the `is_`/`has_` prefixes convention that is present in _most_ of the codebase, what are your thoughts on naming this new method to `is_mandate_data_required`?

Reply:
Sure, makes total sense, thank you! 765db74015e9af7ec324fb34e3d04fdb9ef7ed5a